components/shared/nftCard/NftMediaInfo.vue

```diff
@@ -52,13 +53,25 @@ const props = withDefaults(
     showPrice?: boolean
     collectionPopoverShowDelay?: number
     variant?: NftCardVariant
+    displayNameWithSn?: boolean
   }>(),
   {
     collectionPopoverShowDelay: 500,
     variant: 'primary',
   },
 )
 
+const nameWithSn = computed(() => {
```

Comment:
I would rename it 'name' since it doesnt Always has serial number

Reply:
done